src/main/java/org/assimbly/gateway/authenticate/domain/User.java

```diff
@@ -24,6 +26,8 @@ public class User {
     private Boolean usesTwoFactor;
     private String secretKey;
 
+    protected static Logger log = LoggerFactory.getLogger(User.class);
+
```

Comment:
I later check something about using log classes. They recommend using it like this:

`	protected Logger log = LoggerFactory.getLogger(getClass());
`

Reply:
Yeah, I know, but I want to use log inside the static getEnumFromDocument() method, and the getClass() method is a non-static method.
That's why I used like this.